src/csharp/Microsoft.Spark.E2ETest/UdfTests/BroadcastTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.UdfTests
+{
+    [Serializable]
+    public class BroadcastExampleType
```

Comment:
Let's not use `Type` in the class name. You can name it to `TestBroadcastVar` or something